app/models/vm.rb

```diff
@@ -121,6 +121,12 @@ def supported_consoles
     }
   end
 
+  def v2v_migrated?
+    ServiceResource.where(:resource => self, :status => 'complete').any? do |resource|
```

Comment:
@bzwei Is `complete` a constant somewhere in the transformation models that we can reference?  

I actually think this is should be "Completed" from here: [app/models/service_template_transformation_plan_task.rb#L30](https://github.com/ManageIQ/manageiq/blob/4acfef61b79afa74dd137d8a5675a4b425e13ecb/app/models/service_template_transformation_plan_task.rb#L30).  Which feels like it should really be a constant in the model.